.github/workflows/agentcore_service_test.yml

```diff
@@ -0,0 +1,192 @@
+      - 'sermant-integration-tests/agentcore-test/agentcore-test-application/**'
+      - 'sermant-integration-tests/agentcore-test/agentcore-integration-test/**'
+      - 'agentcore_service_test.yml'
+      - '.github/actions/common/xds-service/**'
+      - '.github/actions/common/agentcore/**'
+      - '.github/actions/scenarios/xds-service/**'
+      - '.github/actions/scenarios/agentcore/dynamic-config/**'
+      - '.github/actions/common/plugin-change-check/action.yml'
+      - '.github/actions/common/entry/action.yml'
+      - '.github/actions/common/exit/action.yml'
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}-${{ github.head_ref }}
+  cancel-in-progress: true
+jobs:
+  set-execution-conditions:
+    name: set-execution-conditions
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 100
+      - name: plugin-change-check
+        id: plugin-change-check
+        uses: ./.github/actions/common/plugin-change-check
+      - name: set-outputs
+        id: set-outputs
+        run: |
+          echo "enableDynamicConfigServicAction=${{env.enableDynamicConfigServicAction}}" >> $GITHUB_OUTPUT
+          echo "enableXdsServicAction=${{env.enableXdsServicAction}}" >> $GITHUB_OUTPUT
+    outputs:
+      enableDynamicConfigServicAction: ${{ steps.set-outputs.outputs.enableDynamicConfigServicAction }}
+      enableXdsServicAction: ${{ steps.set-outputs.outputs.enableXdsServicAction }}
+  download-midwares-and-cache:
+    name: download midwares and cache
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: cache local cse
+        uses: actions/cache@v3
+        with:
+          path: Local-CSE-2.1.3-linux-amd64.zip
+          key: ${{ runner.os }}-local-cse
+          restore-keys: |
+            ${{ runner.os }}-local-cse
+      - name: download cse
+        run: |
+          export ROOT_PATH=$(pwd)
+          bash ./sermant-integration-tests/scripts/tryDownloadMidware.sh cse
+      - name: cache zookeeper
+        uses: actions/cache@v3
+        with:
+          path: apache-zookeeper-3.6.3-bin.tar.gz
+          key: ${{ runner.os }}-apache-zookeeper-3.6.3
+          restore-keys: |
+            ${{ runner.os }}-apache-zookeeper-3.6.3
+      - name: download zookeeper
+        run: |
+          export ROOT_PATH=$(pwd)
+          bash ./sermant-integration-tests/scripts/tryDownloadMidware.sh zk
+      - name: cache nacos server
+        uses: actions/cache@v3
+        with:
+          path: nacos-server-2.1.0.tar.gz
+          key: ${{ runner.os }}-nacos-server-2.1.0
+          restore-keys: |
+            ${{ runner.os }}-nacos-server-2.1.0
+      - name: download nacos
+        run: |
+          export ROOT_PATH=$(pwd)
+          bash ./sermant-integration-tests/scripts/tryDownloadMidware.sh nacos210
+  build-agent-and-cache:
+    name: build agent and cache
+    runs-on: ubuntu-latest
```

Comment:
add a condition judgment about whether this job should be executed

Reply:
done